PythonTool/GrabImagesFromOtherSources.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+import visit as Vi
+
+
+def GrabImagesFromOtherSources(Data):
+	"""Reload sessions with different sources"""
+
+	DataSource = []
+	for item in Data:
+		DataSource.append("../Data/"+item)
+
+	DataSource = tuple(DataSource)
+
+	Directory = str(os.getcwd())+"/../Sessions/XML_Original"
+
+	for file in sorted(os.listdir(Directory)):
+		if file.endswith(".session"):
+			Vi.RestoreSessionWithDifferentSources(
+												  os.path.join(Directory, file),
```

Comment:
I decided to remove that script. It was something experimental I tried but never got to successfully integrate into the overall tool.